data-raw/internalData.R

```diff
@@ -34,6 +34,15 @@ groundColor <- "grey"
 
 sshsPalette <- c("#00CCFF", "#00CCCC", "#FFFFB2", "#FECC5C", "#FD8D3C", "#F03B20", "#BD0026")
 
+SSHS_PALETTE <- c("#00CCFF", "#00CCCC", "#FFFFB2", "#FECC5C", "#FD8D3C", "#F03B20", "#BD0026")
```

Comment:
Why do you need this one, it is already defined just above ?
```suggestion
```

Reply:
I would like to rename all constants in capital letters but you told me that you prefer keeping them in lower case

Reply:
Done